tests/formats/operations/README.md

```diff
@@ -18,11 +18,11 @@ A YAML-encoded `BeaconState`, the state before applying the operation.
 
 Also available as `pre.ssz`.
 
-### `<operation-name>.yaml`
+### `<input-name>.yaml`
```

Comment:
Also note that `input-name` is the more accurate Operations table field used for file naming, though there is only a distinction for `block_header`, which takes a `block.yaml` not a `block_header.yaml`